kong/pdk/log.lua

```diff
@@ -741,33 +768,16 @@ do
   -- @treturn table the request information table
   -- @usage
   -- kong.log.serialize()
+
   if ngx.config.subsystem == "http" then
     function serialize(options)
-      check_phase(PHASES_LOG)
-
       local ongx = (options or {}).ngx or ngx
```

Comment:
here we can use `options = options or {}` like above code.

Reply:
great idea